### ISSUE TITLE
creat a embed code ?

### ISSUE THREAD

**Issue by Deniloee49** [ArdiArtani/Photo-Google-Player-Script#15]
what it's possible to make embed code?

**Comment by Zebraslive** [comment]
https://github.com/Zebraslive/Photo-Google-Player-Script/blob/f157472e5f0e891c75e5a1aec5f39ae8703cd76c/embed.php

**Comment by oceanoffiles** [comment]
hope im not asking too much of you but is it possible to add just the  [ID]/photo/[ID]?key=[Key]. instead of  https://photos.google.com/share/[ID]/photo/[ID]?key=[Key] into the embed,php?id=.

like this embed,php?id=[ID]/photo/[ID]?key=[Key], instead of this  embed,php?id=https://photos.google.com/share/[ID]/photo/[ID]?key=[Key]

**Comment by Amrinder-Ghumman** [comment]
@Zebraslive thanks for the embed code.

**Comment by d4lv** [comment]
@Zebraslive can you make the same embed with jw player 7-8 plzzz

**Comment by Zebraslive** [comment]
@oceanoffiles https://github.com/Zebraslive/Photo-Google-Player-Script/blob/patch-1/embed.php
Done. next time @ me so i know you're talking to me and can answer sooner.
should work both ways now.

@d4lv Sorry I don't use jw player. not a fan of it.

**Comment by d4lv** [comment]
@Zebraslive is any chance to know what is the limit now for streaming files from google photos??

**Comment by cobyperez** [comment]
@Zebraslive Good day I tried the embed code but there's no resolution switcher? can you fix it? sorry but im not a programmer. thanks for the effort.

**Comment by KaranJain29** [comment]
@Zebraslive embed.php is not working. I used this script provided by you https://github.com/Zebraslive/Photo-Google-Player-Script/blob/f157472e5f0e891c75e5a1aec5f39ae8703cd76c/embed.php

But it isn't working. Sample URL:  [https://app.uwatchfreemovies.in/embed.php?=https://photos.google.com/share/AF1QipMTEPAiVF8t0YqLukflnOSQjwfd8ARIoT2h37AXvYO1uaWodbeiFoBUDuD_19tEbg/photo/AF1QipPA2Bq0JlAR9LoGD3mogsxSb9OZWEG4XqBDD4Rv?key=cjhUT0xrZjM5NGN2SVRLOVptZU5SMUlKV0lQYWpB](https://app.uwatchfreemovies.in/embed.php?=https://photos.google.com/share/AF1QipMTEPAiVF8t0YqLukflnOSQjwfd8ARIoT2h37AXvYO1uaWodbeiFoBUDuD_19tEbg/photo/AF1QipPA2Bq0JlAR9LoGD3mogsxSb9OZWEG4XqBDD4Rv?key=cjhUT0xrZjM5NGN2SVRLOVptZU5SMUlKV0lQYWpB)


It says no id provided. Please help

**Comment by karankankaria** [comment]
@Zebraslive  Also when I try to move video forward it justs restarts from begining. I don't know why that error is occuring. Please help.

**Comment by Zebraslive** [comment]
@KaranJain29 Looks like you missed the ?id= in your url. If it restarts from beginning that means it's streaming off the download url which doesn't support Range request header so you can't skip thru the video.

**Comment by karankankaria** [comment]
@Zebraslive Hello Sir. Thanks. I understood the problem. I wanted to ask if the link could be encrypted so that photos..google.com/example cannot be accessed?

**Comment by karankankaria** [comment]
@Zebraslive I have an example for you. This guy has implemented google photos player and he has enabled encryption on it. Link: https://uplayer.stream/demo/photos-panel/

Waiting for your reply!!

**Comment by Zebraslive** [comment]
@KaranJain29 Yes you can encode the URL using openssl. then decode inside gp.php before generating stream links. auto encode the url while passing url to the embed from the database.

**Comment by karankankaria** [comment]
@Zebraslive Hello sir. Thanks for your fast reply. But I am not dev. Can you help me with setting it up. Thanks once again!!

**Comment by karankankaria** [comment]
@d4lv You can check this. https://github.com/KaranJain29/Google-Photo-Player-JW-Player-Integrated
